src/test/java/api/InstancesApiExamples.java

```diff
@@ -308,6 +308,17 @@ public void shouldReturnServerErrorIfOneOfInstancesPostedWithoutTitle() throws M
     assertEquals(postResponse.getJson().getInteger("totalRecords"), Integer.valueOf(2));
   }
 
+  @Test
+  public void shouldReturnBlockedFieldsConfig() throws MalformedURLException, InterruptedException, ExecutionException, TimeoutException {
+    CompletableFuture<Response> getCompleted = new CompletableFuture<>();
+    okapiClient.get(ApiRoot.blockedFieldsConfig(), ResponseHandler.json(getCompleted));
+    Response getResponse = getCompleted.get(5, TimeUnit.SECONDS);
+
+    JsonObject response = getResponse.getJson();
```

Comment:
What happens if the response is a failure, or responds with a different status code?

Reply:
I added checking for proper status code 200